eva_cttv_pipeline/trait_mapping/output.py

```diff
@@ -0,0 +1,77 @@
+import csv
+
+from eva_cttv_pipeline.trait_mapping.trait import Trait
+
+
+def output_trait_mapping(trait: Trait, mapping_writer: csv.writer):
+    """
+    Write any finished ontology mappings for a trait to a csv file writer.
+
+    :param trait: A trait with finished ontology mappings in finished_mapping_set
+    :param mapping_writer: A csv.writer to write the finished mappings
+    """
+    for ontology_entry in trait.finished_mapping_set:
+        mapping_writer.writerow([trait.name, ontology_entry.uri, ontology_entry.label])
+
+
+def get_zooma_mappings_for_curation(trait: Trait) -> list:
+    zooma_entry_list = []
+    for zooma_mapping in trait.zooma_mapping_list:
+        for zooma_entry in zooma_mapping.zooma_entry_list:
+            if zooma_entry.in_efo and zooma_entry.is_current:
+                zooma_entry_list.append(zooma_entry)
+    zooma_entry_list.sort(reverse=True)
+    return zooma_entry_list
+
+
+# TODO best way to unite this and the above very similar method?
```

Comment:
It might make more conceptual sense to change ZoomaMapping to ZoomaResult and ZoomaEntry to ZoomaMapping. And similarly change zooma_entry_list to mapping_list and oxo_mapping_list to mapping_list. Does this sound ok?